tests/conftest.py

```diff
@@ -1961,17 +1964,23 @@ def bucket_factory(request, bucket_class_factory, mcg_obj):
     """
     Returns an MCG bucket factory
```

Comment:
Add to description when this fixture returns None

Reply:
Done